sdk/storage/Azure.Storage.Files.DataLake/src/DataLakeFileClient.cs

```diff
@@ -19,6 +20,12 @@ namespace Azure.Storage.Files.DataLake
     /// </summary>
     public class DataLakeFileClient : DataLakePathClient
     {
+        /// <summary>
+        /// Gets the maximum number of bytes that can be sent in a call
+        /// to <see cref="UploadAsync(Stream, long, PathHttpHeaders, DataLakeRequestConditions, IProgress{long}, StorageTransferOptions, CancellationToken)"/>.
+        /// </summary>
+        public virtual int BlockBlobMaxUploadBlobBytes => Constants.DataLake.MaxUploadBytes;
```

Comment:
if this is public should this be named file instead of blob?

Reply:
@KrzysztofCwalina suggested maybe "MaxUploadBytes"

Reply:
Fixed.